### ISSUE TITLE
Sidebar-menu, missing ul

### ISSUE THREAD

**Issue by Pasley70** [iionly/celebrations#1]
The sidebar-menu-items "Celebrations by the months" should set into an ul-block-element:

```
<ul>
<li class="elgg-menu-item-january"><a href="#" class="elgg-menu-content">January</a></li>
...
```

**Comment by iionly** [comment]
Ah! That explains the dots displayed in Chrome (but not in Firefox...). As I use mainly Firefox I didn't notice for quite some time and when I noticed in Chrome I wondered but postponed to investigate what's wrong.

The fix would have to made in mod/celebrations/views/default/celebrations/sidebar.php:

``` php
echo "<ul>" . $celebrations_monthly . "</ul>";
```

if you want to correct it on your site before I will be able to publish a fixed version.

**Comment by Pasley70** [comment]
Thanks for the hint! Nice Plugin, by the way ;-)

**Comment by iionly** [comment]
Fixed in release 1.9.5.
